pylegend/_typing.py

```diff
@@ -16,6 +16,7 @@
 from typing import Dict as PyLegendDict
 from typing import Iterator as PyLegendIterator
 from typing import List as PyLegendList
+from typing import Set as PyLegendSet
```

Comment:
Why do you alias all the types? You can just use native types since PEP 585 (i.e. `set` instead of `Set`), but I've never seen anybody alias all the types like this.